app/models/host/managed.rb

```diff
@@ -284,7 +284,8 @@ def handle_realm
   # returns the host correct disk layout, custom or common
   def diskLayout
     @host = self
-    pxe_render((disk.empty? ? ptable.layout : disk).gsub("\r",""))
+    template = disk.blank? ? ptable.layout : disk
```

Comment:
Long-standing bug right on this line is that if `ptable.nil?` then you get: `»There was an error rendering the Kickstart default template: «undefined method `layout' for nil:NilClass`, perhaps you'd be kind enough to fix it?